pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -110,55 +110,32 @@ def cli_exit(prog_name: str = 'pai'):  # pragma: no cover
 qualified_model_names = [n for n in get_literal_values(KnownModelName.__value__) if ':' in n]
 
 
-def _handle_version_and_list(
```

Comment:
I reverted just this bit only because the interface was odd. I guess it makes more sense within cli itself